.gitignore

```diff
@@ -4,3 +4,11 @@
 PKG-INFO
 /dist
 /build
+.idea/.name
```

Comment:
This belongs in a [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).